bazel/extension_config/extensions_build_config.bzl

```diff
@@ -358,6 +358,7 @@ ENVOY_CONTRIB_EXTENSIONS = {
 
     "envoy.tls.key_providers.cryptomb":                         "//contrib/cryptomb/private_key_providers/source:config",
     "envoy.tls.key_providers.qat":                              "//contrib/qat/private_key_providers/source:config",
+    "envoy.tls.key_providers.sgx":                              "//contrib/sgx/private_key_providers/source:config",
```

Comment:
You need to add this also to ISTIO_ENABLED_CONTRIB_EXTENSIONS below.

Reply:
Done